CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 14.0.0
+
+- Removed: `stylelint` less than `16.0.0` from peer dependencies.
```

Comment:
nit

Technically `16.0.0-0` is lower than `16.0.0` so it should say _than `16.0.0-0`_.